BAH-515 | Add amman bed management module

sai789 opened the pull request:
https://bahmni.atlassian.net/browse/BAH-515

We forked your repo and added our files and tested on Bahmni DB. Everything working fine.
Two common files changed as part of this. 
1. `ui/app/common/concept-set/mappers/observationMapper.js`
2. `ui/app/common/domain/services/bedService.js`

In `observationMapper.js` file, removed a unused variable in a method.
In `bedService.js` file, added a method `getCompleteBedDetailsByBedId()`

Here are the changed files list:
app/ipd/ --> new folder
app/i18n/ipd/ --> new folder
app/styles/ipd/ --> new folder
/test/unit/ipd/ --> new folder
karma.conf.js --> Include ipd tests

djazayeri commented on ui/app/i18n/ipd/locale_pt_BR.json:
```diff
@@ -0,0 +1,24 @@
+{
```
translations should not be merged in the main product, but these should happen in transifex (like in other apps)

sai789 replied:
We removed this translation file in PR

sai789 commented on ui/app/common/concept-set/mappers/observationMapper.js:
```diff
@@ -54,12 +54,12 @@ Bahmni.ConceptSet.ObservationMapper = function () {
         });
     };
 
-    var mapObservation = function (concept, savedObs, conceptSetConfig, parentConcept) {
```
We have removed extra parameter but we forgot to remove in method calls. Now we removed extra passing parameter wherever it is called.

sai789 commented:
@SwathiVarkala We have addressed these feedbacks. Please have a look at them
1. Gender display in bed management (restricted to M/F) : ipd/bedManagement.html
2. i18 is not done for Gender field on bed management page ipd/bedManagement.html
3. camelCase not used for method, variable names

sai789 commented:
@SwathiVarkala We thought to implement your suggestion. But it seems the createLayoutGrid function implemented with two forLoops which are creating required empty beds, but with this approach we are unable to generate the empty beds.
1. bedManagementService.js createLayoutGrid can be simplified using the below object sort [{x:1,y:2},{x:2,y:1},{x:1,y:3}].sort((a,b) => parseInt(a.x+''+a.y) - parseInt(b.x+''+b.y)) sorting
